### DISCUSSION TITLE
Jamulus 3.8.2_1 *.deb packet does not install on UbuntuStudio 22.04 - missing dependencies

### DISCUSSION THREAD

**Discussion by mrmusixx** [jamulussoftware/jamulus#2623] [Technical Support & Help]
Hi all,
I just tried installing the Jamulus 3.8.2_1 *.deb packet on new Ubuntu Studio 22.04 .
It complains about dependencies that can't be fulfilled:

![image](https://user-images.githubusercontent.com/2129471/166152272-e6d3de1f-7e5a-4145-bef8-71c6de7f49f9.png)

Can someone check, please?

Greetings 

mrmusixx

**Comment by jujudusud** [comment]
Hi here, what does detail say ?

**Comment by mrmusixx** [comment]
Hi there, 
"Details" only shows original package info, size etc. Nothing special about the fault.

However:
As the screenshot implies, the older version 3.8.1-... available from the original Ubuntu Studio 22.04 software channel works fine (and will do for the moment here...). So it's just something about the 3.8.2 from jamulus.io and its packaging.

Thanks

mrmusixx

**Comment by hoffie** [comment]
I've just tried reproducing this on Ubuntu 22.04 in a container. The .deb installs fine there, so the problem might be Ubuntu Studio-specific? It might also be a missing/disabled standard repo and/or an outdated package list.

In order to analyze this further, we would need the actual error message. I don't know if and how it is possible to get it from the GUI, but a Terminal should work:

```
sudo apt update
sudo apt install ~/Downloads/jamulus_3.8.2-1_ubuntu_amd64.deb
```
(You may need to adjust the path to the actual download path of course; we'd need all the output)

**Reply by rdica** [reply]
I spun up a live VM of Ubuntu Studio 22.04 (it uses KDE as default desktop now). Downloaded 3.8.2 deb, ran `sudo apt install ./jamulus-3.8.2_1_ubuntu_amd64.deb`, it installed with a number of apt warnings (mostly due to being run from a live media probably), tried starting it, errored due to not having a jack server running (there wasn't one) which is understandable. There were no dependency issues as part of installation process itself.
